src/sentry/south_migrations/0455_create_global_searches.py

```diff
@@ -0,0 +1,1269 @@
+# -*- coding: utf-8 -*-
+from south.utils import datetime_utils as datetime
+from south.db import db
+from south.v2 import DataMigration
+from django.db import models
+
+
+from sentry.models.savedsearch import DEFAULT_SAVED_SEARCHES
+
+
+class Migration(DataMigration):
+
+    # Flag to indicate if this migration is too risky
+    # to run online and needs to be coordinated for offline
+    is_dangerous = False
+
+    def forwards(self, orm):
+        db.commit_transaction()
+        try:
+            self._forwards(orm)
+        except Exception:
+            db.start_transaction()
+            raise
+        db.start_transaction()
+
+    def _forwards(self, orm):
+        "Write your forwards methods here."
+        # Note: Don't use "from appname.models import ModelName".
+        # Use orm.ModelName to refer to models in this application,
+        # and orm['appname.ModelName'] for models in other applications.
+        SavedSearch = orm['sentry.SavedSearch']
+
+        for search in DEFAULT_SAVED_SEARCHES:
+            SavedSearch.objects.create(
```

Comment:
This won't be idempotent without any UNIQUE index or anything, and this migration isn't done transactionally. Just something to keep in mind.

Reply:
Are you sure it's not inside a transaction? This documentation says that forward/backward is:
https://south.readthedocs.io/en/latest/migrationstructure.html
(unless we have something custom that messes with that)

Reply:
I'm positive because I wrote the behavior change. :) It's really bad for us to do data migrations within a transaction.

Check what is spit out in `forwards`.